stubs/Pygments/pygments/sphinxext.pyi

```diff
@@ -8,14 +8,8 @@ FMTERDOC: str
 FILTERDOC: str
```

Comment:
if you were feeling really fancy, you could type these using `Literal`.

as in,

```python
MODULEDOC: Literal['''
.. module:: %s

%s
%s
''']

LEXERDOC: Literal['''
.. class:: %s

    :Short names: %s
    :Filenames:   %s
    :MIME types:  %s

    %s

    %s

''']

FMTERDOC: Literal['''
.. class:: %s

    :Short names: %s
    :Filenames: %s

    %s

''']

FILTERDOC: Literal['''
.. class:: %s

    :Name: %s

    %s

''']
```

but... that might be overkill

Reply:
it's a thing today! just wouldn't have a great deal of benefit, and would make the type annotations brittle in the sense that any change to the strings would introduce errors.

recommend you disregard this.